frontend/app/utils/loadLocale.ts

```diff
@@ -0,0 +1,17 @@
+/** this is generated file by "npm run generate-langs" **/
+import enMessages from '../locales/en.json';
+
+export async function loadLocale(locale: string): Promise<Record<string, string>> {
+  if (locale === 'ru') {
```

Comment:
Maybe `switch...case`?

Reply:
It is generated code. I don't think that `switch...case` more readable